symfony/client/src/orangehrmTimePlugin/components/ProjectAutocomplete.vue

```diff
@@ -41,6 +41,11 @@ export default {
       required: false,
       default: true,
     },
+    excludeProjectIds: {
+      type: Array,
+      required: false,
+      default: () => [],
+    },
```

Comment:
@super-chama please check this. is there any best way?

Reply:
What's the use case of this? @VageeshOrangeHRM

Reply:
to declare an empty array